tox.ini

```diff
@@ -8,22 +8,28 @@
 # Alternatively, to run mock tests only, run tox as follows:
 # CB_USE_MOCK_PROVIDERS=True tox -e py27-aws
 #
-# Simply running tox -e py27-aws also works, because the default is to use
+# Simply running tox -e py27-aws also works because the default is to use
 # mock providers.
 
 [tox]
-envlist = {py27,py36,pypy}-{aws,openstack}
+envlist = {py36}-{azure}
```

Comment:
@VikramAIS If we comment this out though, the tests won't run for aws and openstack either, and the py27 tests aren't in the list, so we need to fix that. I think Travis supports a flag to enforce PyPy versions, so we should be able to work around the cryptography issue (the GCE provider is also running into the same issue). My suggestion is to restore everything and add azure. I'll put in a fix for the crypto issue when merging.